### DISCUSSION TITLE
2022/02/07/Screeps%E5%85%A5%E9%97%A8%E6%95%99%E7%A8%8B3-%E5%8D%87%E7%BA%A7%E8%80%85/

### DISCUSSION THREAD

**Discussion by giscus** [Westlifers/Westlifers.github.io#14] [Announcements]
# 2022/02/07/Screeps%E5%85%A5%E9%97%A8%E6%95%99%E7%A8%8B3-%E5%8D%87%E7%BA%A7%E8%80%85/

本文将介绍房间等级，控制器，最终实现一个智能升级者 房间等级（RCL, Room Controller Level）每个房间都有对应的等级，顾名思义，房间等级实际是由一个控制器决定，我们马上就会提到。房间等级决定了房间拥有者在此房间可以使用的武器/建筑。 控制器（controller）在你的新房间中，有一个闪着微弱红光的结构，这是房间的控制器。控制器有等级，体现在周围一圈的白色分段进度条上。而控制

https://hexo.yougi.top/2022/02/07/Screeps%E5%85%A5%E9%97%A8%E6%95%99%E7%A8%8B3-%E5%8D%87%E7%BA%A7%E8%80%85/

**Comment by ghost** [comment]
透过你深邃的文字,我仿佛看到了你鹰视狼顾,龙行虎步的伟岸英姿;仿佛看到了你手执如椽大笔,写天下文章的智慧神态;仿佛看见了你按剑四顾,江山无数的英武气概!你说的多好啊

**Comment by ghost** [comment]
你的语言浅显易懂，你的思想却深入骨髓，我对你的敬仰之情犹如滔滔江水连绵不绝。。
